### ISSUE TITLE
Umbrel lnd channels all closed, unable to connect to node

### ISSUE THREAD

**Issue by sahilc0** [getumbrel/umbrel#1362]
Hi there, not sure what changed but all of a sudden I'm unable to log into Thunderhub, and all my lightning channels are offline.

Here's a debug log - seems like there's some errors at the end. Any idea what could be the issue?
[umbrel-1651027827135-debug.log](https://github.com/getumbrel/umbrel/files/8568337/umbrel-1651027827135-debug.log)

**Comment by sahilc0** [comment]
Also worth noting I've tried restarting both Umbrel and lnd specifically, no luck.

**Comment by sahilc0** [comment]
LNDg doesn't load at all: 
![image](https://user-images.githubusercontent.com/8256062/165551633-f1f0f061-e900-4feb-acac-10bcb779d3fe.png)

**Comment by sahilc0** [comment]
Update on what I've tried:

**1. Tried resetting lnd.conf**
`rm ~/umbrel/lnd/lnd.conf && ~/umbrel/scripts/configure && sudo reboot`

**2. Confirmed lnd is running** 
with `docker-compose ps`

So doesn't seem to be a lnd.conf issue.

**3. Checked lnd logs**
Lots of "RPC services not available" 
![image](https://user-images.githubusercontent.com/8256062/165667293-8c041a00-3a76-4418-ad0a-a7e922f79184.png)
![image](https://user-images.githubusercontent.com/8256062/165667336-b2786438-ef16-4c8c-a6a3-29bda29ba695.png)
![image](https://user-images.githubusercontent.com/8256062/165667533-77fb0018-496a-4b7d-942b-adc027a32536.png)
^(even though it seems to have fetched the latest block, at least according to the dashboard UI)
**Seems like there's some issue here?**

**4. Checking if wallet.dat file exists**
`ls -la -h ~/umbrel/bitcoin/`
Seems like no sign of wallet.dat file
![image](https://user-images.githubusercontent.com/8256062/165667712-7b352d11-1bce-44aa-95e0-8dfa5ce9e4c3.png)

At a loss for what to do beyond this point. Seems to be something with:
- "RPC services not available" and
- wallet.dat file not being there

**Comment by sahilc0** [comment]
Update: I do have a wallet.db file. 

Still no luck.

**Comment by louneskmt** [comment]
Any idea @guggero? 🙏

**Comment by mayankchhabra** [comment]
Sorry for the issue, Sahil! While I'll wait for the experts to chime in, the missing `wallet.dat` file is a non-issue since a wallet isn't created in Bitcoin Core by either LND or us. The on-chain wallet you see on the Bitcoin page is essentially LND's on-chain wallet.

From your attached logs, the only issue seems to be that LND is not being able to unlock the wallet (which is causing all the other issues):

```
lnd                  | 2022-04-27 02:48:21.319 [ERR] RPCS: [/lnrpc.Lightning/ListChannels]: wallet locked, unlock it to enable full RPC access
lnd                  | 2022-04-27 02:48:21.323 [ERR] RPCS: [/lnrpc.Lightning/GetInfo]: wallet locked, unlock it to enable full RPC access
lnd                  | 2022-04-27 02:48:22.478 [ERR] RPCS: [/lnrpc.Lightning/GetInfo]: wallet locked, unlock it to enable full RPC access
lnd                  | 2022-04-27 02:48:22.549 [ERR] RPCS: [/lnrpc.Lightning/GetInfo]: wallet locked, unlock it to enable full RPC access
lnd                  | 2022-04-27 02:48:22.899 [ERR] RPCS: [/lnrpc.Lightning/GetInfo]: wallet locked, unlock it to enable full RPC access
```

**Comment by guggero** [comment]
Can you paste the full `lnd` log?
It looks like it can't be unlocked (I assume that's supposed to be done automatically on Umbrel?). The `block not found` error doesn't look good, maybe something with bitcoind's block index?

**Comment by sahilc0** [comment]
@mayankchhabra True! I don't have indication that it's locked from the GUI though, and I tried lncli unlock on cli but no luck there either. 

@guggero will do ASAP. Unfortunately my entire building lost power for a week or more, so I'm going to have to get back to you guys later :(

**Comment by mpnemo** [comment]
On the latest update, you can't run lncli via the docker, its depreciated
use 
go to your home directory, ususally
/home/umbrel/umbrel
and urn
 ./scripts/app compose lightning exec lnd lncli <command>
